### PR TITLE
apache-jena-fuseki: 4.8.0 -> 4.9.0, add a test

### PR DESCRIPTION
**Pull request by 7c6f434c**
###### Description of changes

Adding a minimal test showing the Fuseki starts and handles a small insert and a request. As the test is run in the minimal environment, add the missing utilities to PATH in the wrapper.

Related to report by Joeri Exelmans about Fuseki not starting as-is in minimal environments.

Related to #244442 

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [x] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by LeSuisse — APPROVED** [review]
Changes looks good and tests pass.

File: pkgs/servers/nosql/apache-jena/fuseki-test.nix [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{ lib, runCommandNoCC, apache-jena-fuseki, curl }:
```
Comment (dotlambda) [inline pkgs/servers/nosql/apache-jena/fuseki-test.nix:1]:
```
{ lib, runCommand, apache-jena-fuseki, curl }:
```
This is an alias and breaks eval. Please fix asap!

Comment (dotlambda) [reply pkgs/servers/nosql/apache-jena/fuseki-test.nix:1]: > Does this need fixing in staging-next or master?

master

Comment (7c6f434c) [reply pkgs/servers/nosql/apache-jena/fuseki-test.nix:1]: https://github.com/NixOS/nixpkgs/pull/247699

**Pull request merged**